### ISSUE TITLE
No reaction from calibre GUI when trying to open OverDrive Libby

### ISSUE THREAD

**Issue by krair** [ping/libby-calibre-plugin#13]
Installed the plugin through the GUI, and copied my 8 digit code into the plugin settings. When trying to open the GUI within Calibre, nothing happens. 

Reloading Calibre in debug mode gives me a tiny bit of info in the debug log:

```
Traceback (most recent call last):
  File "/usr/lib64/calibre/calibre/gui2/ui.py", line 223, in initialize
    ac.do_genesis()
  File "/usr/lib64/calibre/calibre/gui2/actions/__init__.py", line 145, in do_genesis
    self.genesis()
  File "calibre_plugins.overdrive_libby.action", line 76, in genesis
    image_resources = get_resources(
                      ^^^^^^^^^^^^^^
TypeError: get_resources() got an unexpected keyword argument 'print_tracebacks_for_missing_resources'
calibre Debug log
calibre 5.43  embedded-python: False is64bit: True
Linux-6.4.14-200.fc38.x86_64-x86_64-with-glibc2.37 Linux ('64bit', 'ELF')
('Linux', '6.4.14-200.fc38.x86_64', '#1 SMP PREEMPT_DYNAMIC Sat Sep  2 16:36:06 UTC 2023')
Python 3.11.5
```
As you can see, Linux desktop app, Fedora 38, Calibre 5.43.

Am I missing something?

**Comment by ping** [comment]
Thanks for the report.

Can you test if [overdrive-libby-plugin-f6ff126 .zip](https://github.com/ping/libby-calibre-plugin/suites/16073482052/artifacts/917488977) fixes it for you?

**Comment by ping** [comment]
I was able to test the fix with a portable version of 5.34 and things seem working.

This fix has also been incorporated into the latest release 0.1.9.
